src/test/java/unit/Workspace.java

```diff
@@ -116,6 +116,29 @@ void statusDescribeListReflectDelete() throws IOException {
     assertEquals(0, matchingWorkspaces.size(), "deleted workspace is not included in list");
   }
 
+  @Test
+  @DisplayName("delete property")
+  void deleteProperty() throws IOException {
+    TestUser testUser = TestUser.chooseTestUserWithSpendAccess();
+    testUser.login();
+    String initialProperties = "key=value,key1=value1,foo=bar";
+
+    // create a workspace with 3 properties
+    WorkspaceUtils.createWorkspace(testUser, "propertyDeleteTest", "", initialProperties);
+
+    // call `terra workspace delete-property` for 2 properties
+    TestCommand.runCommandExpectSuccess("workspace", "delete-property", "--keys=key,key1");
+
+    UFWorkspace describeWorkspace =
+        TestCommand.runAndParseCommandExpectSuccess(UFWorkspace.class, "workspace", "describe");
+    // check workspace only has 1 property
```

Comment:
nit usually there is a blank line before this kind of comment. The comment explains the code after it. So put line 134 on line 132.

```
    // comment
    some code

    // comment
    some code
```